app/cells/decidim/problems/problem_m/show.erb

```diff
@@ -0,0 +1,54 @@
+<div class="card card--problem <%= resource_state %> muted">
+    <div class="card__content">
+        <div class="card__header">
+            <%= link_to problem_path, class: "card__link" do %>
+                <div class="card__title">
+                    <%= resource_title %>
+                </div>
+            <% end %>
+        </div>
+        <%#= cell("decidim/coauthorships", problem, has_actions: false, context: { current_user: current_user }) %>
+        <div class="card__text--problem">
+            <div class="card__text--paragraph">
+                <span class="card__text--paragraph--challenge"><%= t("challenge", scope: 'decidim.problems') %>: </span>
+                <%= link_to challenge_path, class: "card__link card__text--paragraph--challenge--title" do %>
+                    <div><%= resource_challenge_title%></div>
+                <% end %>
+            </div>
+        </div>
+        <div class="card__text">
+            <div class="card__text--paragraph">
+                <span class="card__text--paragraph--estat"><%= t(resource_state, scope: 'decidim.problems.states') %></span>
```

Comment:
This should be rendered with all letters upcase, in challenges I got it rendered as normal text, but should be as follows in both challenges and problems:
![imatge](https://user-images.githubusercontent.com/199462/100548873-b4bd5e00-326f-11eb-97db-012fea1f03b3.png)

Reply:
Also use always English in the code. Rename `card__text--paragraph--estat` please